rails-time-track/db/seeds.rb

```diff
@@ -5,3 +5,72 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require 'faker'
+
+3.times do
+  User.create(
+    name: Faker::Name.name,
+    email: Faker::Internet.email,
+    # role: Faker::Job.position,
+    role: "Owner",
+    rate: Faker::Number.between(1000,2000)
```

Comment:
It could help us if we set a fixed rate (to make calculations in the future)

Reply:
oh right. We have a calculated value.